source/addComment.js

```diff
@@ -1,3 +1,25 @@
 $(document).ready(function(){
-  // add your code here
-})
+
+  $('#new_comment_button').on('click', function () {
+      event.preventDefault();
+
+    $('#new_comment_button').hide()
+    $( '<form id="new_comment" name="new_comment" style="display: block"></form>' ).appendTo( "body" );
```

Comment:
You could also consider `.hide()` and `.show()` for this form, and just add the conntent to the HTML file.